packages/client/tests/functional/issues/13766/prisma/_schema.ts

```diff
@@ -11,25 +11,21 @@ export default testMatrix.setupSchema(({ provider }) => {
     datasource db {
       provider = "${provider}"
       url      = env("DATABASE_URI_${provider}")
-    }
-
-    enum OrderStatus {
-      NEW
-      PICKED
-      STORED
-      DELIVERED
+      referentialIntegrity = "prisma"
     }
 
     model Order {
-      orderId          ${idForProvider(provider)}
+      id               ${idForProvider(provider)}
+      orderId          String @unique
       paid             Boolean?
       statusMilestones OrderStatusHistory[]
     }
 
     model OrderStatusHistory {
-      orderStatusHistoryId ${idForProvider(provider)}
+      id                   ${idForProvider(provider)}
+      orderStatusHistoryId String
```

Comment:
Why do we need 2 IDs here @Jolg42? Workaround for test setup limitation?

Reply:
Workaround for testing MongoDB since the _id is immutable

Reply:
Maybe worth separating into two different tess then? One for Mongo with this setup, and one that really reproduces the original issue? I am not sure what different behavior this could trigger. Or are you sure enough it does not?

Reply:
see new commit and README, I split into 2 schemas.

Reply:
Nice!